.gitignore

```diff
@@ -91,4 +91,5 @@ modules.xml
 # Sonarlint plugin
 .idea/sonarlint
 
-# End of https://www.toptal.com/developers/gitignore/api/clion+all
\ No newline at end of file
+# End of https://www.toptal.com/developers/gitignore/api/clion+all
+/.vs
```

Comment:
This shouldn't be part of this PR as it seems unrelated to the subject of this PR.